modules/mint/handle_block.go

```diff
@@ -3,40 +3,12 @@ package mint
 import (
 	juno "github.com/forbole/juno/v2/types"
 
-	"github.com/rs/zerolog/log"
 	tmctypes "github.com/tendermint/tendermint/rpc/core/types"
-
-	"github.com/forbole/bdjuno/v2/types"
 )
 
 // HandleBlock implements modules.BlockModule
 func (m *Module) HandleBlock(
```

Comment:
Since we don't need block handling for mint module anymore, should we remove it?

Reply:
removed ^^